vmdb/app/models/ems_refresh/save_inventory.rb

```diff
@@ -17,7 +17,7 @@ def save_vms_inventory(ems, hashes, target = nil)
     log_header = "MIQ(#{self.name}.save_vms_inventory) EMS: [#{ems.name}], id: [#{ems.id}]"
 
     disconnects = if target.kind_of?(ExtManagementSystem) || target.kind_of?(Host)
-      target.vms_and_templates(true).dup
+      target.vms_and_templates(true).to_a.dup
```

Comment:
@kbrock it won't *hurt* anything to blindly change them.  But we don't know if we actually *need* to.  This particular case is expecting to:

1. Eagerly load the association and have it in memory
2. Add a record to the association, but not in the eagerly loaded collection
3. Update the eagerly loaded records **and not** the newly created one

I'm not sure if all the other places actually do that.  Maybe the do?